docs/dyndns/update-api.rst

```diff
@@ -100,3 +104,40 @@ Update Response
 If successful, the server will return a response with status ``200 OK`` and
 ``good`` as the body (as per the dyndns2 protocol specification). For error
 status codes, see above.
+
+dynDNS updates are subject to rate limiting.  For details, see
+:ref:`rate-limits`.
+
+
+Examples
+````````
+
+The examples below use ``<your domain>.dedyn.io`` as the domain which is to be updated and 
+``<your authorization token>`` as an API token affiliated with the respective account.
+(See :ref:`manage-tokens` for details.) ``<1.2.3.4>`` is used as an example for an IPv4 Address, 
+``<fd08::1234>`` as a standin for an IPv6 address. Replace those (including the ``<`` and ``>``)
+with your respective values.
+
+
+Basic authentication with automatic IP detection (IPv4 **or** IPv6)::
+
+  curl --user <your domain>.dedyn.io:<your authorization token> https://update.dedyn.io
+  
+  curl https://update.dedyn.io/?hostname=<your domain>.dedyn.io \ 
+    --header "Authorization: Token <your authorization token>"
+
+Basic authentication with forced use of IPv6 (will remove IPv4 address from the DNS)::
+
+  curl --user <your domain>.dedyn.io:<your authorization token> https://update6.dedyn.io
+  
+  curl https://update6.dedyn.io/\?hostname\=<your domain>.dedyn.io \ 
+    --header "Authorization: Token <your authorization token>"
```

Comment:
Let's avoid escaping backslashes, e.g. by quoting the URL. (I think it's platform-dependent where the backslashes would have to go, but quotes should work everywhere.)

Reply:
this should be fixed.